models/news.model.js

```diff
@@ -39,39 +36,50 @@ const readArticlesID = (id) =>{
       for(let i = 0; i < result.rows.length; i++ ) {
         Object.assign(finalObj, result.rows[i]);
       }
-     
       return finalObj
       }
-      })
-    
-
- 
-
+      }) 
 }
 
 const readComments = (req) =>{
 
   const { article_id } = req.params;
-  
-  return db
+    return db
     .query(
       "SELECT * FROM comments WHERE article_id = $1 ORDER BY created_at DESC;",
       [article_id]
     )
     .then((result) => {
+      
       if (result.rows.length === 0){
         return Promise.reject({status: 404, msg: "Article not found"});
       } else{
       return result.rows;
       }
+   
     })
-    
 
+}
 
+function addComment(req) {
+  const { article_id } = req.params;
+  return db
+    .query(
+      "INSERT INTO comments (body, author, article_id, votes)VALUES($1,$2,$3,$4) RETURNING *;",
+      [req.body[0]["body"], req.body[0]["username"], article_id, 0]
+    )
+    .then((result) => {
+      
+    return result.rows;
```

Comment:
You can take the comment off rows and return it back to the controller here, as per my other comments.